commands/team_test.go

```diff
@@ -898,8 +902,10 @@ func (s *MmctlUnitTestSuite) TestRestoreTeamsCmd() {
 			Times(1)
 
 		err := restoreTeamsCmdF(s.client, cmd, []string{"team1"})
-		s.Require().Nil(err)
-		s.Require().Equal("Unable to restore team 'team1' error: an error occurred restoring a team",
-			printer.GetErrorLines()[0])
+		var expected error
+		expected = multierror.Append(expected, errors.New("Unable to restore team '"+teamName+"' error: an error occurred restoring a team"))
+
+		s.Require().NotNil(err)
```

Comment:
Same here

Reply:
Removed check and used string interpolations for error formatting.